app/pages/stacking/direct-stacking/components/choose-btc-address.tsx

```diff
@@ -56,6 +56,12 @@ export const ChooseBtcAddressStep: FC<ChooseBtcAddressStepProps> = props => {
   const form = useFormik({
     initialValues: { btcAddress: '' },
     validate: ({ btcAddress }) => {
+      const isValid = validate(btcAddress);
```

Comment:
name conflict?

Reply:
No, it's fine as the other validate is only an object prop name